### ISSUE TITLE
Allow users to register their own Handlebars helpers

### ISSUE THREAD

**Issue by jeffywu** [OpenZeppelin/solidity-docgen#213]
Would be nice to allow users to register their own handlers, the current set of handlers is quite sparse.

**Comment by frangio** [comment]
Hi @jeffywu! What do you mean by handlers exactly?

**Comment by jeffywu** [comment]
It would be useful to be able to add custom functionality to the handlebars templates via handlers:
https://github.com/OpenZeppelin/solidity-docgen/blob/master/src/handlebars.ts

I have use cases for trimming strings or filtering the list of functions, etc.

**Comment by frangio** [comment]
Ah I see, that's interesting. I think it makes sense.

Something like this?

`solidity-docgen --helpers helpers.js`

```
// helpers.js
module.exports = {
  trim(str) { return ... },
  ...
};
```

**Comment by jeffywu** [comment]
Yeah that would be perfect

**Comment by frangio** [comment]
@jeffywu I've added the `--helpers` option as described in the comment above.

Could you try it out in the prerelease and let me know how it goes before I publish it?

`npm install solidity-docgen@next`

Thank you!

**Comment by jeffywu** [comment]
Hi I'll take a look on Sunday, I'm taking a couple days off but will get back to you. Thanks for the quick turnaround!

**Comment by frangio** [comment]
Great! I've published it as `0.5.4`.
